bin/import-wp-css-storybook.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+STORYBOOK_WORDPRESS_DIR="$DIR/../storybook/wordpress";
+STORY_BOOK_CSS_PATH="$DIR/../storybook/wordpress/css";
+TMP_DIR="$DIR/../storybook/wordpress/tmp";
+ARCHIVE_CSS_PATH="wordpress/wp-admin/css";
+ARCHIVE_IMG_PATH="wordpress/wp-admin/images";
+
+mkdir -p $STORY_BOOK_CSS_PATH;
+mkdir -p $TMP_DIR;
+
+function download_and_extract_css {
```

Comment:
This basic script runs before storybook and downloads the latest Wordpress to extract the needed styles. Unfortunately the CSS references images so we have to extract those too to avoid the build breaking.

Reply:
@joshuatf I think we will need to keep this for a while to make the docs look nice. In future my hope is that we would refactor and improve our components to remove the need for this, but I don't expect this to happen quickly.